File: Backend/src/main/java/com/mycompany/myapp/service/dto/VentaAsientosRequest.java
package com.mycompany.myapp.service.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.time.Instant;
import java.time.LocalDateTime;
import java.util.List;
@Data
@AllArgsConstructor
@NoArgsConstructor
public class VentaAsientosRequest implements Serializable {
    private Long eventoId;
    private Instant fecha;
    private Double precioVenta;
    private List<AsientosVentasDTO> asientos;
}

File: Frontend/composeApp/src/commonMain/kotlin/org/example/project/dto/VentaAsientosRequest.kt
package org.example.project.dto

import kotlin.time.Instant
import kotlinx.serialization.Serializable

@Serializable
data class VentaAsientosRequest(
    val eventoId: Long,
    val fecha: String,
    val precioVenta: Double,
    val asientos: List<AsientosComprasDTO>
)
